### PR TITLE
docs: fix duplicate subpage listing

### PR DESCRIPTION
**Pull request by gsquared94**
Fixes: #6481

### CODE REVIEW

**Comment by MarlonGamez** [comment]
I think this is an interesting one because our manually created lists don't seem to match up 100%, but the auto generated ones do. Do you think we should continue with the manual lists and try to maintain them?

**Comment by briandealwis** [comment]
I'm with @MarlonGamez — let Hugo/docsy do the heavy lifting

**Comment by gsquared94** [comment]
> I'm with @MarlonGamez — let Hugo/docsy do the heavy lifting

done 👍

File: docs/content/en/docs/tutorials/_index.md [hunk 1]
```diff
@@ -22,10 +23,3 @@ When deploying to a remote cluster you have to point Skaffold to your default im
 ### What's next
```
Comment (briandealwis) [inline docs/content/en/docs/tutorials/_index.md:23]: It's odd seeing a `what's next` before the actual content.  TBH, I think we should:

1. turn the default-repo into a callout `{{ <alert> }}` — but really this should be incorporated into each of the tutorial pages
2. move the survey into the footer shown on each page
2. get rid of most of the remaining text

**Review by briandealwis — COMMENTED** [review]
Thanks for cleaning this up.  I have some more suggestions 😱

File: docs/content/en/docs/tutorials/_index.md (hunk 1 above)
Comment (gsquared94) [reply docs/content/en/docs/tutorials/_index.md:23]: ya, those definitely seem out of place

Comment (gsquared94) [reply docs/content/en/docs/tutorials/_index.md]: how about now?
http://34.94.56.85:1313/docs/tutorials/

moved Survey to footer and named it `Feedback`
image

**Comment by gsquared94** [comment]
> I think this is an interesting one because our manually created lists don't seem to match up 100%, but the auto generated ones do. Do you think we should continue with the manual lists and try to maintain them?

@MarlonGamez updated where feasible to use auto list, PTAL.

**Review by MarlonGamez — APPROVED** [review]
Looks good to me!

**Pull request merged**